diagnostics/test/main.cpp

```diff
@@ -29,7 +31,7 @@ int main() {
   registrar.registerStatusHandler(handler2);
 
   concord::diagnostics::Server diagnostics_server;
-  diagnostics_server.start(registrar);
+  diagnostics_server.start(registrar, INADDR_LOOPBACK, PORT);
```

Comment:
Did you test that this works with container to container communication? I thought we need to use the docker bridge device or `INADDR_ANY`.

Reply:
Nope, this is just a test main that you have to run manually anyway. I only use it for validating that ./concop works as expected. The change in the product will pass in INADDR_ANY